Dependency issue - build broke

Hi, thanks for sharing this repository, this is awesome!

I'm trying to run `mvn clean install` and am getting this error:
```
[ERROR] Failed to execute goal on project searoute: Could not resolve dependencies for project eu.europa.ec.eurostat:searoute:war:1.3-SNAPSHOT: The following artifacts could not be resolved: org.geotools:gt-main:jar:22.3, org.geotools:gt-graph:jar:22.3, org.geotools:gt-shapefile:jar:22.3, org.geotools:gt-cql:jar:22.3, org.geotools:gt-referencing:jar:22.3, org.geotools:gt-epsg-hsql:jar:22.3, org.geotools:gt-epsg-extension:jar:22.3, org.geotools:gt-geojson:jar:22.3, org.geotools:gt-render:jar:22.3, org.geotools:gt-swing:jar:22.3, org.geotools:gt-brewer:jar:22.3, org.geotools:gt-geopkg:jar:22.3: Failure to find org.geotools:gt-main:jar:22.3 in http://download.osgeo.org/webdav/geotools/ was cached in the local repository, resolution will not be reattempted until the update interval of osgeo has elapsed or updates are forced
```

Btw, your docs seem to be outdated, the root of the project no longer contains you `pom.xml`. (I also tried installing the project version at commit `b98dc243a789e34dafb047e480de35d401a22cf5`, just in case. Same error).

Cheers!

--- a/modules/pom.xml
+++ b/modules/pom.xml
@@ -59,9 +59,11 @@
 
 	<repositories>
 		<repository>
-			<id>osgeo</id>
-			<name>Open Source Geospatial Foundation Repository</name>
-			<url>http://download.osgeo.org/webdav/geotools/</url>
+		    <id>osgeo</id>
+		    <name>OSGeo Release Repository</name>
+		    <url>https://repo.osgeo.org/repository/release/</url>
+		    <snapshots><enabled>false</enabled></snapshots>
+		    <releases><enabled>true</enabled></releases>
 		</repository>
 	</repositories>
 

